app/javascript/react/screens/App/Overview/components/Migrations/Migrations.js

```diff
@@ -143,14 +144,14 @@ const Migrations = ({
           )}
           {activeFilter === MIGRATIONS_FILTERS.archived && (
             <MigrationsCompletedList
-              finishedTransformationPlans={archivedTransformationPlans}
+              finishedTransformationPlans={Immutable.asMutable(archivedTransformationPlans)}
```

Comment:
[Since we're not sorting archived plans](https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/427#issuecomment-402180597), we can remove the `asMutable` here, no?